functions/index.js

```diff
@@ -28,3 +29,73 @@ app.use((err, req, res, next) => {
     });
 });
 exports.api = functions.region('europe-west1').https.onRequest(app);
+exports.signUpEmailNotification = functions.region('europe-west1')
+    .firestore.document('users/{userId}').onCreate(async snapshot => {
+        try {
+            const { email, firstName, lastName, userId } = snapshot.data();
+            const receipent = email;
+            const subject = 'Welcome to NeoNatar';
+            const text = `Dear ${firstName} ${lastName},
+        reach out to all your esteemed customers suing our platform`;
```

Comment:
I think the suing should be using

Reply:
why are these strings here?

Reply:
It is a temporary text sent as an email notification. Pending when we design a better UI page for email notification.